app/utils/PropertyDetailsUtils.scala

```diff
@@ -23,6 +23,9 @@ import uk.gov.hmrc.http.InternalServerException
 
 object PropertyDetailsUtils extends ReliefConstants {
 
+  val _2012ValuationPolicyDate = Some(new LocalDate("2012-04-01"))
+  val _2017ValuationPolicyDate = Some(new LocalDate("2017-04-01"))
```

Comment:
these can be worked out from the period you fall into

Reply:
Yes they could be - these are just constants used in a few places in the file..